test/unit/commands/recordscreen-specs.js

```diff
@@ -0,0 +1,135 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import AndroidDriver from '../../..';
+import _ from 'lodash';
+import { withMocks } from 'appium-test-support';
+import ADB from 'appium-adb';
+import { sleep } from 'asyncbox';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('recording the screen', () => {
+  let driver = new AndroidDriver();
+  let adb = new ADB();
+  driver.adb = adb;
+
+  describe('recording the screen', withMocks({adb, driver}, (mocks) => {
```

Comment:
I think that 
the recording function is out of business that 
making an emulator environment in unit test case, and checking whether it is in emulator environment in unit test case.